qt/applications/workbench/workbench/widgets/settings/test_utilities/settings_test_utilities.py

```diff
@@ -0,0 +1,18 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench
+from inspect import getmembers
+from unittest.mock import call
+
+from qtpy.QtWidgets import QComboBox, QSpinBox, QDoubleSpinBox
+
+
+def test_presenter_has_added_mousewheel_filter_to_all_como_and_spin_boxes(view, mock_mousewheel_filter):
```

Comment:
```suggestion
def test_presenter_has_added_mousewheel_filter_to_all_combo_and_spin_boxes(view, mock_mousewheel_filter):
```

Reply:
Changing the first part of this function name from `test_` to `assert_` might make it clearer this is a utility function rather than a test in itself?